conans/errors.py

```diff
@@ -68,7 +68,9 @@ class ConanException(Exception):
     """
          Generic conans exception
     """
-    pass
+    def __init__(self, *args, **kwargs):
+        self.info = None
```

Comment:
No one is setting the info yet?

Reply:
Not currently, only adjusted in already created exceptions, so I prefer to keep it as it is to avoid confusions.